web/styles/message_row.css

```diff
@@ -470,6 +470,14 @@ $time_column_min_width: 42px; /* + padding */
         display: unset !important;
         cursor: default;
     }
+
+    /* Mesage-editing styles */
```

Comment:
"Mesage" is missing an `s`. I'm a bit surprised that the `./tools/run-codespell` in CI didn't catch this.

Reply:
Weird. I'll push a fix